packages/renderer/src/lib/image/ImageActions.svelte

```diff
@@ -64,8 +52,7 @@ async function deleteImage(): Promise<void> {
   try {
     await imageUtils.deleteImage(image);
   } catch (error) {
-    errorTitle = 'Error while deleting image';
-    errorMessage = String(error);
+    onError(`Error while deleting image: ${error}`);
```

Comment:
Before it was `String(error)` so i would keep it the same -> `Error while deleting image:  ${String(error)}`